src/utils/proxyLazy.ts

```diff
@@ -54,7 +57,12 @@ handler.getOwnPropertyDescriptor = (target, p) => {
     if (typeof p === "string" && unconfigurable.includes(p))
         return Reflect.getOwnPropertyDescriptor(target, p);
 
-    return Reflect.getOwnPropertyDescriptor(target.get(), p);
+    if (p !== GET_KEY && p !== CACHED_KEY) return;
```

Comment:
```suggestion
    if (p === GET_KEY || p === CACHED_KEY) throw "NO";
```

is this what u meant? Otherwise it makes no sense to me

Reply:
it was to prevent it from breaking is someone defined those symbol props on a module, but i make the keys unique symbols to make it not possible